### PR TITLE
README: how to connect via psql

### PR DESCRIPTION
**Pull request by hjacobs**
Readers should see the whole flow and know how to connect to their new test cluster in Minikube :smile:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,7 +134,17 @@ permissions there).
...
 ### Watch Pods being created
 
     $ kubectl --context minikube  get pods -w --show-labels
+
+### Connect to PostgreSQL
+
+We can use the generated secret of the `postgres` robot user to connect to our `acid-testcluster` master running in Minikube:
+
+    $ export HOST_PORT=$(minikube service acid-testcluster --url | sed 's,.*/,,')
+    $ export PGHOST=$(echo $HOST_PORT | cut -d: -f 1)
```
Comment (hjacobs) [inline README.md]: I open for suggestions on how to improve this shell magic..

**Comment by ikitiki** [comment]
👍

File: README.md (hunk 1 above)
Comment (ikitiki) [inline README.md]: you can use ``minikube ip`` to get the ip address

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**